collectors/gsuite/cfn/gsuite_collector.template

```diff
@@ -0,0 +1,111 @@
+            "Type": "String",
+            "Default": ""
+        },
+        "CollectorId": {
+            "Description": "Optional. A collector UUID if known.",
+            "Type": "String",
+            "Default": "none"
+        },
+        "PawsEndpoint": {
+            "Description": "URL to poll",
+            "Type": "String"
+        },
+        "PawsAuthType": {
+            "Description": "Target API authentication type. Supported types: ssws, oauth2",
+            "Type": "String",
+            "Default": "ssws"
+        },
+        "PawsClientId": {
+            "Description": "Client ID for oauth2 authentication type",
+            "Type": "String"
+        },
+        "GsuiteClientSecret": {
+            "Description": "Gsuite Client secret",
+            "Type": "String",
+            "NoEcho": true
+        },
+	    "GsuiteScope": {
+            "Description": "Define Scope, if it is multiple pass as comma separated string for Gsuite",
+            "Type": "String"
+        },
+        "GsuiteApplicationNames": {
+            "Description": "Define Application name, if it is multiple pass as comma separated string for Gsuite",
+            "Type": "String"
+        },
+        "GsuiteEmailId": {
+            "Description": "Email ID for Gsuite",
+            "Type": "String"
+        },
+        "CollectionStartTs": {
+            "Description": "Optional. Timestamp when log collection starts. For example, 2019-11-21T16:00:00Z",
+            "Type": "String",
+            "AllowedPattern" : "(?:^\\d{4}(-\\d{2}){2}T(\\d{2}:){2}\\d{2}Z$)?"
+        }
+    },
+    "Resources":{
+      "GsuiteCollectorStack" : {
+        "Type" : "AWS::CloudFormation::Stack",
+        "Properties" : {
+          "TemplateURL" : {"Fn::Join" : ["", [
+                    "https://s3.amazonaws.com/",
+                    {"Ref":"PackagesBucketPrefix"}, "-us-east-1",
+                    "/cfn/paws-collector.template"
+               ]]},
+          "Parameters" : {
+            "AlertlogicAccessKeyId" : { "Ref":"AlertlogicAccessKeyId" },
+            "AlertlogicSecretKey" : { "Ref":"AlertlogicSecretKey" },
+            "AlApiEndpoint" : { "Ref":"AlApiEndpoint" },
+            "AlDataResidency" : { "Ref":"AlDataResidency" },
+            "PackagesBucketPrefix" : { "Ref":"PackagesBucketPrefix" },
+            "PawsCollectorTypeName" : "Gsuite",
+            "AlertlogicCustomerId" : { "Ref":"AlertlogicCustomerId" },
+            "CollectorId" : { "Ref":"CollectorId" },
+            "PollingInterval" : 60,
+            "PawsEndpoint" : { "Ref":"PawsEndpoint" },
+            "PawsAuthType" : { "Ref":"PawsAuthType" },
+            "PawsClientId" : "",
+            "PawsSecret" : { "Ref":"GsuiteClientSecret" },
+            "PawsScope" : { "Ref":"GsuiteScope" },
+            "PawsApplicationNames" : { "Ref":"GsuiteApplicationNames" },
```

Comment:
This does not exist on the base template. This should be "CollectorParamString2". you should also reference i this way in your code.